match/lib/match/storage/git_storage.rb

```diff
@@ -31,7 +34,7 @@ def configure(type: nil,
         self.git_url = git_url
         self.shallow_clone = shallow_clone
         self.manual_password = manual_password
-        self.skip_docs = skip_docs # TODO: We don't use this yet
+        self.skip_docs = skip_docs # We don't use this yet
```

Comment:
Ah I need to look into those, thanks

Reply:
Np! I Made this a separate PR so that the todo removals stood out more 🙃